eth/vm/code_stream.py

```diff
@@ -15,20 +15,24 @@
 class CodeStream(object):
     stream = None
     depth_processed = None
+    _len = None
+    _value = None
 
     logger = logging.getLogger('eth.vm.CodeStream')
 
     def __init__(self, code_bytes: bytes) -> None:
         validate_is_bytes(code_bytes, title="CodeStream bytes")
         self.stream = io.BytesIO(code_bytes)
+        self._value = code_bytes
+        self._len = len(code_bytes)
```

Comment:
Thoughts on renaming these to:

- `_len > _length_cache`
- `_value > _raw_code_bytes` or `_code_bytes`

Reply:
way more descriptive yes :)